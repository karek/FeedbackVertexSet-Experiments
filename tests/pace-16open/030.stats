40,292,27
