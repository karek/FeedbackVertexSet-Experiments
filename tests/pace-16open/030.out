19
23 32 21 9 38 20 14 37 39 36 13 6 8 7 4 0 28 25 35 
